data/json/mapgen/zoo.json

```diff
@@ -57,6 +57,93 @@
+      "rows": [
+        "----------------------ss",
+        "-__________________|vsss",
+        "-__________________|vsss",
+        "-__|||||||||||||||||ssss",
+        "-__|...{{..........wssss",
+        "-__|...............wssss",
+        "-__|......{.{.||ww||+|+|",
+        "-__|......{.{.|.........",
+        "-__|..c.......+.........",
+        "-__|..c.......|.........",
+        "-__|.Cc......&w.........",
+        "-__|ccc...{..Hw....|||++",
+        "-__|......{..Hw....|__ss",
+        "-__|.........H|....|f_ss",
+        "-__|||||||||||||..||D_ss",
+        "-__|cS|hh.r..&|....|b_ss",
+        "-__|..d...r...|...#|f_ss",
+        "-__|..|...r.......#|__ss",
+        "-__|..|^h.r.......#r__ss",
+        "-__|..||||r..#|...#r__ss",
+        "-__|..|...r..#|...#r__ss",
+        "-__|..d...r...|...#r__ss",
+        "-__|..|0..r..#|...#|__ss",
+        "-__|..|i..r..#|....|__ss"
+      ],
+      "terrain": {
+        "+": "t_reinforced_door_glass_c",
+        "-": "t_brick_wall",
+        "D": "t_dirt",
+        "_": [ "t_bamboo", "t_bamboo_long", "t_bamboo_tall", "t_grass" ],
+        "b": "t_dirt",
+        "d": "t_door_metal_c",
+        "f": "t_dirt",
+        "r": "t_reinforced_glass",
+        "s": "t_sidewalk",
+        "v": "t_sidewalk",
+        "w": "t_reinforced_glass",
+        ".": "t_floor",
+        "|": "t_brick_wall"
+      },
+      "furniture": {
+        "#": "f_bench",
+        "&": "f_trashcan",
+        "0": "f_boulder_medium",
+        "C": "f_chair",
+        "D": "f_dandelion",
+        "H": "f_sofa",
+        "S": "f_sink",
+        "^": "f_indoor_plant_y",
+        "b": "f_bluebell",
+        "c": "f_counter",
+        "f": "f_dahlia",
+        "h": "f_hay",
+        "i": "f_indoor_plant",
+        "v": "f_vending_c",
+        "{": "f_rack"
+      },
+      "place_item": [ { "item": "rock", "repeat": 1, "x": 9, "y": 15 }, { "item": "pine_bough", "repeat": 1, "x": 9, "y": 21 } ],
+      "place_items": [
+        { "chance": 55, "item": "toy_store", "x": 12, "y": [ 6, 7 ], "repeat": [ 1, 16 ] },
+        { "chance": 75, "item": "vending_food", "x": 20, "y": 2 },
+        { "chance": 55, "item": "trash", "x": 13, "y": 15 },
+        { "chance": 55, "item": "trash", "x": 13, "y": 10 },
+        { "chance": 55, "item": "snacks", "x": 10, "y": 11, "repeat": [ 1, 16 ] },
+        { "chance": 55, "item": "snacks", "x": [ 7, 8 ], "y": 4, "repeat": [ 1, 16 ] },
+        { "chance": 55, "item": "candy_shop", "x": 10, "y": [ 6, 7 ], "repeat": [ 1, 16 ] },
+        { "chance": 75, "item": "shirts", "x": 10, "y": 12, "repeat": [ 1, 5 ] },
+        { "chance": 45, "item": "shirts", "x": 4, "y": 11, "repeat": [ 1, 3 ] },
+        { "chance": 75, "item": "vending_drink", "x": 20, "y": 1 },
+        { "chance": 25, "item": "waitingroom", "x": 18, "y": [ 16, 22 ], "repeat": [ 1, 3 ] }
+      ],
+      "place_monster": [
+        { "monster": "mon_wolf", "x": 8, "y": 16 },
+        { "monster": "mon_wolf", "x": 9, "y": 17 },
```

Comment:
Could you rather use the "GROUP_WOLFS" in here? So they can spawn with wolf pups in the zoo? Sad to see them go from the wild, but well, at least they remain in here.